simple-todo/package.json

```diff
@@ -4,22 +4,23 @@
   "author": "Paul Adams",
   "private": true,
   "scripts": {
-    "dev": "webpack-dev-server --inline --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
+    "dev": "webpack-dev-server --mode=development --inline --hot",
```

Comment:
`npm run dev` complains a mode is not set. I attempted to leverage cross-env, and left in some features within webpack.config.js to leverage those. However, I have never had much luck using cross-env over the builtin flag.